lang-ca_valencia.js

```diff
@@ -0,0 +1,1394 @@
+
+    // collision detection
+    'mouse-pointer':
+        'punter del ratolí',
+    'edge':
+        'vora',
+    'pen trails':
+        'rastre del llapis',
+
+    // costumes
+    'Turtle':
+        'Tortuga',
+    'Empty':
+        'Buit',
+
+    // graphical effects
+    'brightness':
+        'brillantor',
+    'ghost':
+        'fantasma',
+    'negative':
+        'negatiu',
+    'comic':
+        'còmic',
+    'confetti':
+        'confeti',
+
+    // keys
+    'space':
+        'espai',
+    'any key':
+        'qualsevol tecla',
+    'up arrow':
+        'fletxa amunt',
+    'down arrow':
+        'fletxa avall',
+    'right arrow':
+        'fletxa dreta',
+    'left arrow':
+        'fletxa esquerra',
+    'a':
+        'a',
+    'b':
+        'b',
+    'c':
+        'c',
+    'd':
+        'd',
+    'e':
+        'e',
+    'f':
+        'f',
+    'g':
+        'g',
+    'h':
+        'h',
+    'i':
+        'i',
+    'j':
+        'j',
+    'k':
+        'k',
+    'l':
+        'l',
+    'm':
+        'm',
+    'n':
+        'n',
+    'o':
+        'o',
+    'p':
+        'p',
+    'q':
+        'q',
+    'r':
+        'r',
+    's':
+        's',
+    't':
+        't',
+    'u':
+        'u',
+    'v':
+        'v',
+    'w':
+        'w',
+    'x':
+        'x',
+    'y':
+        'y',
+    'z':
+        'z',
+    '0':
+        '0',
+    '1':
+        '1',
+    '2':
+        '2',
+    '3':
+        '3',
+    '4':
+        '4',
+    '5':
+        '5',
+    '6':
+        '6',
+    '7':
+        '7',
+    '8':
+        '8',
+    '9':
+        '9',
+
+    // messages
+    'new...':
+        'nou...',
+
+    // math functions
+    'abs':
+        'valor absolut',
+    'ceiling':
+        'sostre',
+    'floor':
+        'part entera',
+    'sqrt':
+        'arrel quadrada',
+    'sin':
+        'sin',
+    'cos':
+        'cos',
+    'tan':
+        'tan',
+    'asin':
+        'asin',
+    'acos':
+        'acos',
+    'atan':
+        'atan',
+    'ln':
+        'ln',
+    'e^':
+        'e^',
+
+    // delimiters
+    'letter':
+        'lletra',
+    'whitespace':
+        'espai en blanc',
+    'line':
+        'línia',
+    'tab':
+        'tabulador',
+    'cr':
+        'retorn de carro',
+
+    // data types
+    'number':
+        'nombre',
+    'text':
+        'text',
+    'Boolean':
+        'Booleà',
+    'list':
+        'llista',
+    'command':
+        'ordre',
+    'reporter':
+        'reportador',
+    'predicate':
+        'predicat',
+
+    // list indices
+    'last':
+        'últim',
+    'any':
+        'qualsevol',
+
+    // attributes
+    'neighbors':
+        'veïns',
+    'self':
+        'un mateix',
+    'other sprites':
+        'els altres objectes',
+    'parts':
+        'parts',
+    'anchor':
+        'àncora',
+    'parent':
+        'pare',
+    'children':
+        'fill',
+    'clones':
+        'clons',
+    'other clones':
+        'altres clons',
+    'dangling?':
+        'Baumeln?',
```

Comment:
Açò no me pareix pas valencià ;)

Reply:
Well, that's the Stuttgart variant of the Valencian variant of Catalan :-D

Reply:
There are so many German tourists that you guys are starting to influence the language!

@joamuran com ho traduiries tu? En @jguille2 ha posat "penjat?" en valencià del nord :)